tools/repl_test.py

```diff
@@ -86,6 +86,19 @@ def test_multiline(self):
         assertEqual(err, '')
         assertEqual(code, 0)
 
+    def test_set_timeout(self):
+        # Special treatment
+        p = Popen([self.deno_exe], stdout=PIPE, stderr=PIPE, stdin=PIPE)
+        # Print after 0.1 second
+        p.stdin.write(
+            "setTimeout(() => console.log('HI'), 100)\n".encode("utf-8"))
+        sleep(0.2)  # Wait 0.2 second before proceed
```

Comment:
This could be done with the existing `input` method, without "special treatment" 😄, and avoid the repetition. I will make a separate PR.